kv/txn_coord_sender.go

```diff
@@ -758,28 +756,26 @@ func (tc *TxnCoordSender) heartbeat(id string) {
 					Reply: reply,
 				}
 
-				task := tc.stopper.StartTask()
-				if !task.Ok() {
+				if !tc.stopper.RunTask(func() {
```

Comment:
I'll check this in as is and look into how we might remove the boolean return. It's definitely its own PR/discussion. This PR as such is an improvement IMHO with or without the return code.